src/main/java/com/carina/methods/saucedemo/commons/pages/CartScreenBase.java

```diff
@@ -0,0 +1,26 @@
+package com.carina.methods.saucedemo.commons.pages;
+
+import com.carina.methods.saucedemo.android.pages.CartScreen;
+import com.carina.methods.saucedemo.android.pages.CheckoutScreen;
+import org.openqa.selenium.WebDriver;
+
+public abstract class CartScreenBase extends AbstractSauceDemoScreenBase {
+
+    public CartScreenBase(WebDriver driver) {
+        super(driver);
+    }
+
+    public abstract boolean isOpened();
+
+
+    public abstract String getCostByName(String productName);
+
```

Comment:
isOpened is already in `AbstractSauceDemoScreenBase`, can be removed here
```suggestion
    public abstract String getCostByName(String productName);

```